scipy/optimize/_differentialevolution.py

```diff
@@ -588,8 +774,8 @@ def _mutate(self, candidate):
 
         fill_point = self.random_number_generator.randint(0, parameter_count)
 
-        if (self.strategy == 'randtobest1exp'
-                or self.strategy == 'randtobest1bin'):
```

Comment:
In that case my PEP8 program makes warnings. I also like it on the continuation line. Can someone confirm that PEP8 test will be passed at Travis CI, if a logical operator will be on the continuation line?